### ISSUE TITLE
[1.18.2]Exit the world/game,the machine does not work

### ISSUE THREAD

**Issue by Tki-sor** [Frinn38/Custom-Machinery#74]
![2023-01-08_15 08 31](https://user-images.githubusercontent.com/78548154/211184925-d34db7fd-ab22-4a86-a953-41a5d32e61a6.png)
Then, exit the world
![2023-01-08_15 08 58](https://user-images.githubusercontent.com/78548154/211184968-ab57aad4-40a9-4cb2-afc9-f9cb4ca6ae0b.png)
But,if exit game,join the world
![2023-01-08_15 15 14](https://user-images.githubusercontent.com/78548154/211184996-68f119e0-dd11-4e84-a2e2-b7a932055d52.png)
Sorry, I don't speak English.

My machine:
[evaporator.zip](https://github.com/Frinn38/Custom-Machinery/files/10367892/evaporator.zip)

**Comment by Frinn38** [comment]
Your machine does not define a model for when it's errored so it use the default model

```json
"appearance": {
      "errored": {}, <------ HERE
      "running": {
        "block":"custommachinery:machine/evaporator_on"
      },
      "idle": {
        "block": "custommachinery:machine/evaporator"
      },
      "paused": {}
}
```

**Comment by Tki-sor** [comment]
> Your machine does not define a model for when it's errored so it use the default model
> 
> ```json
> "appearance": {
>       "errored": {}, <------ HERE
>       "running": {
>         "block":"custommachinery:machine/evaporator_on"
>       },
>       "idle": {
>         "block": "custommachinery:machine/evaporator"
>       },
>       "paused": {}
> }
> ```

I'm sorry for that, but I mean it can be a stable trigger error.
The machine does not retain progress when exiting the game. Just as I did above.


https://user-images.githubusercontent.com/78548154/211196385-580f7057-e6d9-4d59-8799-12685752e6ad.mp4

**Comment by Frinn38** [comment]
Can I see your recipe too ?

**Comment by Tki-sor** [comment]
> 

onEvent('recipes', event => {
event.recipes.custommachinery.custom_machine("unrealworld_machines:evaporator", 3600)
        .requireBlock(["minecraft:fire", "minecraft:campfire"], true, 0, -1, 0, 0, -1, 0)
        .requireFluid('minecraft:water')
        .requireItem('unrealworld:inpure_mineral_salt_dust')
        .produceItem('unrealworld:inpure_nitrate_dust');
})

**Comment by Frinn38** [comment]
Ok I see the problem, there is indeed a bug and it will be fixed for the next update.
Thanks for reporting that bug.

**Comment by Frinn38** [comment]
Forgot to close this issue, that bug was fixed in a previous update (0.7.5)
